internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -13,9 +13,11 @@ import (
 )
 
 var (
-	resourceName = "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive.test"
-	projectID    = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-	endpointID   = os.Getenv("MONGODB_ATLAS_PRIVATE_ENDPOINT_ID")
+	resourceName       = "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive.test"
+	projectID          = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+	endpointID         = os.Getenv("MONGODB_ATLAS_PRIVATE_ENDPOINT_ID")
+	defaultComment     = "Terraform Acceptance Test"
```

Comment:
knitpit: not sure why we say "default..." instead of just comment or region / atlasRegion

Reply:
changed in f4bd5e4 ✅